homeassistant/components/wallbox/__init__.py

```diff
@@ -94,6 +98,24 @@ def _set_charging_current(self, charging_current: float) -> None:
                 raise InvalidAuth from wallbox_connection_error
             raise ConnectionError from wallbox_connection_error
 
+    def _set_lock_unlock(self, lock: bool) -> None:
+        """Set wallbox to locked or unlocked."""
```

Comment:
Maybe move the two new methods below `async_set_charging_current` to keep it together with `_set_charging_current`?

Reply:
Yes, I was actually thinking about that before (keeping async with async, or async with the corresponding sync function, I changed it to the latter now)